src/BloomExe/Collection/BloomPack/BloomPackInstallDialog.cs

```diff
@@ -50,6 +50,7 @@ private void BeginInstall()
 			using (var zip = new ZipFile(_path))
 			{
 				_folderName = GetRootFolderName(zip);
+				zip.Close();
```

Comment:
It appears the using could end here?
I'm not sure the Close() is needed at all...isn't the point of the using to Dispose of the zip, which will typically close it?